### PR TITLE
Uninstall dba sql watch fixes

### PR DESCRIPTION
**Pull request by Ant-Green**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [x] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [x] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
 
To ensure that all objects of SQLWatch are cleaned up correctly

### Approach

Changed wildcard lookups to find the right objects
Added in some new functions - I hope I have done the .Tests.ps1 files correctly I ran manual.pester.ps1 it advised no tests to run, I probably done something wrong.

### Commands to test

install-dbasqlwatch -sqlinstance <> -database sqlwatch
uninstall-dbasqlwatch -sqlinstance <> -database sqlwatch

### CODE REVIEW

**Comment by Ant-Green** [comment]
Got there eventually, any questions on this please fire away

File: functions/Uninstall-DbaSqlWatch.ps1 [hunk 1]
```diff
@@ -90,10 +90,19 @@ Function Uninstall-DbaSqlWatch {
...
+            $funcs += Get-DbaDbUdf -SqlInstance $server -Database $Database | Where-Object { $PSItem.Name -eq "ufn_sqlwatch_get_threshold_comparator" }
+            $agentJobs = Get-DbaAgentJob -SqlInstance $server | Where-Object { $PSItem.Name -like "SqlWatch-*" }
+            $XESessions = Get-DbaXESession -SqlInstance $server | Where-Object { $PSItem.Name -like "SQLWATCH_*" }
+            $Assemblies = Get-DbaDbAssembly -SqlInstance $server -Database $Database | Where-Object { $PSItem.Name -like "SqlWatch*" }
+            $TableTypes = Get-DbaDbUserDefinedTableType -SqlInstance $server -Database $Database | Where-Object { $PSItem.Name -like "utype_*" }
+            $BrokerQueues = Get-DbaDbServiceBrokerQueue -SqlInstance $server -Database $Database | Where-Object { $PSItem.Name -like "sqlwatch_*" }
+            $BrokerServices = Get-DbaDbServiceBrokerService -SqlInstance $server -Database $Database | Where-Object { $PSItem.Name -like "sqlwatch_*" }
+            #Put XE at the end as it messes with the call to Assemblies for some reason
```
Comment (andreasjordan) [inline functions/Uninstall-DbaSqlWatch.ps1:104]: comment not needed anymore

Comment (Ant-Green) [reply functions/Uninstall-DbaSqlWatch.ps1]: Thanks, have removed the comment now.

**Comment by Ant-Green** [comment]
Failing AppVeyor not related to changes in this branch, can someone re-kick it off?

**Review by andreasjordan — APPROVED** [review]

**Comment by andreasjordan** [comment]
> Thank you both 🥳 @andreasjordan is this good to merge?

@potatoqualitee yes, looks very good.

**Comment by potatoqualitee** [comment]
awesome! thanks again 😊

**Pull request merged**